### PR TITLE
Create application page (UI)

### PR DESCRIPTION
**Pull request by olhasashchuk**
# Create an application page (UI)

---

## Description

1. Created Application Page
2. Implemented Overview side
3. Created layout, desktop/mobile version

P.S. 
1. This page does not have the fetch function. It will update after approving this PR: [](https://github.com/FurkannOzbek/application-hub/pull/27)
2. Code about to get data was commented

---

## Changes Made

- **Frontend:** 
1. CustomDrawer.js;
2. DashboardApplicationHeader.js
3. DashboardApplicationMain.js
4. Overview.js

The components will be implemented later: Interview.js, Notes.js, Documents.js, Notes.js

---

## Screenshots (if applicable)
Dasktop Dark mode
Dasktop Light mode
Mobile Light mode
Mobile Dark mode

### CODE REVIEW

File: frontend/app/user/applications/[id]/DashboardApplicationHeader.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  //   )
+  // }
+  // if (error) {
+  //   return (
+  //     <Alert severity="error" sx={{ m: 4 }}>
+  //       {error}
+  //     </Alert>
+  //   )
+  // }
+  // if (!application) return <Typography>No data available.</Typography>
```
Comment (pankaj28843) [inline frontend/app/user/applications/[id]/DashboardApplicationHeader.js:33]: Please delete commented out code

File: frontend/app/user/applications/[id]/Overview.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  useTheme,
+  useMediaQuery,
+} from '@mui/material'
+//import { useApplicationById } from '@/app/hooks/useApplicationById'
+
+const Overview = () => {
+  const params = useParams()
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'))
+  // const id = params.id
```
Comment (pankaj28843) [inline frontend/app/user/applications/[id]/Overview.js:24]: Please delete commented out code.

**Review by pankaj28843 — CHANGES_REQUESTED** [review]

File: frontend/app/user/applications/[id]/DashboardApplicationHeader.js (hunk 1 above)
Comment (olhasashchuk) [reply frontend/app/user/applications/[id]/DashboardApplicationHeader.js]: I removed it.

File: frontend/app/user/applications/[id]/Overview.js (hunk 2 above)
Comment (olhasashchuk) [reply frontend/app/user/applications/[id]/Overview.js]: I removed it.

**Review by pankaj28843 — APPROVED** [review]

**Review by Natata08 — APPROVED** [review]

**Pull request merged**